src/Components/StoryBookExampleComponent/StoryBookExampleComponent.stories.jsx

```diff
@@ -1,124 +1,123 @@
-import { useState } from 'react'
```

Comment:
Why eslint removing single qoutes? It should not do that. We using single qoutes in this project.

Reply:
I've found out, that in code editor eslint wasn't turned on..